source/_posts/2019-07-17-release-96.markdown

```diff
@@ -189,6 +189,7 @@ Experiencing issues introduced by this release? Please report them in our [issue
 - __Velbus__ - Added autodicovery for covers in velbus and bumped the python velbus version ([@Cereal2nd] - [#24877]) ([velbus docs])
 - __ZHA__ - ZHA entities are being removed and battery sensors are being added. We have been asked several times to remove the ZHA domain entities because they do not follow the HA standards. They were originally used to show device online / offline status, rssi, lqi and last seen. They were eventually expanded to hold battery information as well because many Zigbee devices have several power configuration clusters. Online and offline handling, rssi, lqi last seen and the handling of device availability have all been moved to the device object and battery information has been split back out into its own sensor. Any users that created template sensors that leveraged the data in the ZHA entities will be affected by this change. ([@dmulcahey] - [#24909]) ([zha docs])
 - __Netatmo__ - If the module type can not be determined the climate device will default to the type "valve" as it implements a subset of "thermostat". Consequently those devices wont be able to be turned off. This should not be a major issue and should only happen when there are issues with the data retrieved form the Netatmo API. ([@cgtobi] - [#25029]) ([netatmo docs])
+__Netatmo__ - The configuration of individual monitored conditions has been removed so that now all conditions are available per module by default. (@cgtobi - #24944) (netatmo docs)
```

Comment:
```suggestion
- __Netatmo__ - The configuration of individual monitored conditions has been removed so that now all conditions are available per module by default. (@cgtobi - #24944) (netatmo docs)
```